features/rails_features/breadcrumbs.feature

```diff
@@ -25,18 +25,19 @@ Scenario Outline: Request breadcrumb
 
   Examples:
     | ruby_version | rails_version |
-    | 2.0          | 3             |
-    | 2.1          | 3             |
-    | 2.2          | 3             |
-    | 2.2          | 4             |
-    | 2.2          | 5             |
-    | 2.3          | 3             |
-    | 2.3          | 4             |
-    | 2.3          | 5             |
-    | 2.4          | 3             |
-    | 2.4          | 5             |
-    | 2.5          | 3             |
-    | 2.5          | 5             |
+   # | 2.0          | 3             |
```

Comment:
Did you intend to comment these out? (same question applies to features/rails_features/handled.feature)